randovania/data/json_data/prime3.txt

```diff
@@ -1137,28 +1227,103 @@ Asset id: 3467893814575167461
   > Dock to Grand Court Path
       Trivial
   > Dock to Reliquary II
+      After Gateway Tunnel Orb
+  > Base Floor
+      After Gateway Orange Lock and After Gateway Blue Lock
+  > Event - Tunnel Orb
+      Grapple Lasso
+
+> Base Floor; Heals? False
+  > Dock to Crash Site
+      Trivial
+  > Dock to Gateway Hall
+      Trivial
+  > Dock to Reliquary II
+      Use Screw Attack (Space Jump)
+  > Tunnel Room
+      After Gateway Orange Lock and After Gateway Blue Lock
+  > Event - Orange Lock
+      Trivial
+  > Event - Blue Lock
+      After Gateway Orange Lock
+  > Event - Crash Site Side Platform
       Grapple Lasso
 
+> Event - Orange Lock; Heals? False
+  * Event Gateway Orange Lock
+  > Base Floor
+      Trivial
+
+> Event - Blue Lock; Heals? False
+  * Event Gateway Blue Lock
+  > Base Floor
+      Trivial
+
+> Event - Tunnel Orb; Heals? False
+  * Event Gateway Tunnel Orb
+  > Tunnel Room
+      Trivial
+
+> Event - Crash Site Side Platform; Heals? False
+  * Event Gateway Platform near Crash Site
+  > Base Floor
+      Trivial
```

Comment:
This event doesn't seem to have a purpose nor does it link to anything relevant aside from Base Floor. Should move the Lasso requirement to Base Floor -> Blue Lock as that's the intended path.